CommonData/databaseinterface.cpp

```diff
@@ -2174,11 +2176,17 @@ void DatabaseInterface::load()
 	if(!loadingWorked)
 	{
 		if(loadingAttempt > 10)
```

Comment:
Maybe change this to 
```suggestion
		if(loadingAttempt > 100)
```

to see if longer waiting fixes it.

It will print a lot but we can fix that later.